prow/scripts/cluster-integration/helpers/kyma-cli.sh

```diff
@@ -14,6 +15,8 @@ install::kyma_cli() {
 
     curl -sSLo kyma "https://storage.googleapis.com/kyma-cli-stable/kyma-${os}?alt=media"
     chmod +x kyma
+    kyma_version=$(kyma version --client)
+    log::info "Kyma version: ${kyma_version}"
```

Comment:
This will print the client version. To prevent any misunderstandings I'd print `Kyma CLI version:` instead.